.github/workflows/bi-connectors.yml

```diff
@@ -1,6 +1,8 @@
 name: Build connectors for BI tools
 
 on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
So what happens when someone pushes additional commits to an existing PR?

Reply:
It is not triggered anymore if commit has no changes according to path spec.
I just wanted to avoid all workflows being triggered twice on each push. I used this tip from [there](https://github.com/open-telemetry/opentelemetry-python/issues/1370).

Reply:
Gotcha.